example/ipsec_api/Makefile.am

```diff
@@ -2,18 +2,24 @@ include $(top_srcdir)/example/Makefile.inc
 
 bin_PROGRAMS = odp_ipsec_api
 
-dist_check_SCRIPTS = \
-		   odp_ipsec_api_run_ah_in.sh \
-		   odp_ipsec_api_run_ah_out.sh \
-		   odp_ipsec_api_run_ah_tun_in.sh \
-		   odp_ipsec_api_run_ah_tun_out.sh \
-		   odp_ipsec_api_run_esp_in.sh \
-		   odp_ipsec_api_run_esp_out.sh \
-		   odp_ipsec_api_run_esp_tun_in.sh \
-		   odp_ipsec_api_run_esp_tun_out.sh \
-		   odp_ipsec_api_run_live.sh \
-		   odp_ipsec_api_run_router.sh \
-		   odp_ipsec_api_run_simple.sh
+if test_example
+TESTS = \
+	odp_ipsec_api_run_esp_out.sh \
+	odp_ipsec_api_run_simple.sh
```

Comment:
Since these tests are really short you could add most of them here to increase CI testing coverage.

There seems to be something wrong with `odp_ipsec_api_run_esp_in.sh` and `odp_ipsec_api_run_esp_tun_in.sh` tests as they run indefinitely, but that's unrelated to this PR.

Reply:
Ack.

In v7, enabling all the tests except:
* ESP inbound tests
* Live an Router mode tests